code/game/turfs/simulated/water.dm

```diff
@@ -0,0 +1,13 @@
+/turf/open/water
```

Comment:
That one does literally nothing and has a different icon to fit with the beach anyway